zone/botspellsai.cpp

```diff
@@ -3456,3 +3483,17 @@ bool Bot::HasBotSpellEntry(uint16 spellid) {
 
 	return false;
 }
+
+bool Bot::IsValidSpellRange(uint16 spell_id, Mob const* tar) {
+	if (!IsValidSpell(spell_id)) {
+		return false;
+	}
+
+	if (tar) {
+		int spellrange = .8 * (GetActSpellRange(spell_id, spells[spell_id].range) * GetActSpellRange(spell_id, spells[spell_id].range));
```

Comment:
.8 ?

Reply:
Per Nite.
![image](https://user-images.githubusercontent.com/109764533/219896430-b2b3e93d-ad0f-4358-abbf-57994032ac0b.png)

I'll do some testing, and see why this is needed, or least assign it to a local variable to give it more verbosity..

Reply:
Tested myself, and this wasn't required. so removed.